gravitee-plugin-core/src/main/java/io/gravitee/plugin/core/api/AbstractConfigurablePluginManager.java

```diff
@@ -106,7 +131,12 @@ private String getFirstFile(String pluginId, String directory) throws IOExceptio
 
     @Override
     public PluginMoreInformation getMoreInformation(String pluginId) throws IOException {
-        T plugin = get(pluginId);
+        return getMoreInformation(pluginId, false);
+    }
+
+    @Override
+    public PluginMoreInformation getMoreInformation(String pluginId, boolean includeNotDeployed) throws IOException {
+        T plugin = get(pluginId, includeNotDeployed);
         Map<String, String> properties = plugin.manifest().properties();
```

Comment:
Shouldn't we handle the case where no plugin is found ?

Reply:
Yep was not done before but make sense